app/src/main/java/com/nervousfish/nervousfish/modules/cryptography/EncryptorAdapter.java

```diff
@@ -36,4 +64,132 @@ private EncryptorAdapter(final IServiceLocator serviceLocator) {
         return new ModuleWrapper<>(new EncryptorAdapter(serviceLocator));
     }
 
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public String hashWithoutSalt(final String pass) throws EncryptionException {
```

Comment:
any string. The use case is that you can check whether or not a password is correct, without anyone outside of the user being able to read the password. If you save an encrypted string like this using your password and encrypt that same password like this later, it will hash to the same string, meaning you entered the same password.